Tests/Imagine/Filter/Loader/FloatToIntCastByRoundUpscaleFilterLoaderTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Liip\ImagineBundle\Tests\Filter;
+
+use Imagine\Gd\Image;
+use Imagine\Gd\Imagine;
+use Liip\ImagineBundle\Imagine\Filter\Loader\UpscaleFilterLoader;
+use Liip\ImagineBundle\Tests\AbstractTest;
+
+/**
+ * @covers Liip\ImagineBundle\Imagine\Filter\Loader\UpscaleFilterLoader
+ *
+ * Due to int casting in Imagine\Image\Box which can lead to wrong pixel
+ * numbers ( e.g. float(201) casted to int(200) ). Solved by round the
```

Comment:
the phrase `float(201) casted to int(200)` does not look correct. float 201 is cast to int 201. there must be a decimal part which we loose. something like: `float(200.7) casted to int(200)`. 

Could you confirm that?